packages/client/components/ReflectionGroup/DraggableReflectionCard.tsx

```diff
@@ -89,15 +92,17 @@ const DraggableReflectionCard = (props: Props) => {
     meetingId,
     teamId,
     staticReflectionCount,
+    isInSpotlight,
     swipeColumn
   )
+
   const isDragPhase = phaseType === 'group' && !isComplete
   const canDrag = isDraggable && isDragPhase && !isEditing && !isDropping
   // slow state updates can mean we miss an onMouseDown event, so use isDragPhase instead of canDrag
   const handleDrag = isDragPhase ? onMouseDown : undefined
   return (
     <DragWrapper
-      ref={(c) => (drag.ref = c)}
+      ref={(c) => (drag.ref = isBehindSpotlight ? null : c)}
```

Comment:
This fixed a tricky bug. When dragging a reflection group in the Spotlight, `useDraggableReflectionCards` would use the ref that's in the kanban instead of the Spotlight. This made dragging, grouping, and ungrouping very buggy. 

Remote reflections still begin from the kanban position which I think is preferable as it indicates that that's where the teammate is dragging the card from